common-test/src/main/java/io/strimzi/test/KafkaCluster.java

```diff
@@ -23,11 +23,17 @@
     String name();
     int kafkaNodes() default 3;
     int zkNodes() default 1;
+    CmConfiguration[] cmConfiguration() default {};
 
 
     @Target({ElementType.METHOD, ElementType.TYPE})
     @Retention(RetentionPolicy.RUNTIME)
     @interface Container {
         KafkaCluster[] value();
     }
+
+    @interface CmConfiguration {
```

Comment:
I think this would be better as a top level annotation, rather than an inner annotation because it can then be reused in, for example, `ConnectCluster`. And maybe calling it `CmData` would be better (since it specifies stuff to go in a CM's `data` section).